tests/test_utils.py

```diff
@@ -0,0 +1,206 @@
+from lxml import objectify
+import pytest
+from django.forms import ValidationError
+
+from controlled_vocabularies import admin, views, vocabulary_handler
+from factories import PropertyFactory, TermFactory, VocabularyFactory, OrderedTermFactory
```

Comment:
We will want to change this to an explicit relative import (i.e. `.factories`). Also, it looks like we are importing all of the factories from it, so would you be opposed to changing it to

```python
from . import factories
```

?